configurations/dockerDataTemplate.go

```
@@ -117,3 +121,27 @@ func (cmdObj cmd) WriteInstruction() string {
 	result := fmt.Sprintf("CMD %s", execFormString)
 	return result
 }
+
+// Programming Language installation instruction.
+type programmingLanguage struct {
+	Name    string
+	Version string
+}
+
+// Gives RUN instruction for installing the specified programmingLanguage.
+func (langObj programmingLanguage) WriteInstruction() string {
+
+	scriptName := fmt.Sprintf("%s_%s.sh", langObj.Name, langObj.Version)
+
+	// Check whether the given language and version are available in the installation scripts.
+	currentDir, _ := os.Getwd()
```

Again, we should not ignore the error.

---

Fixed